src/Control/Distributed/Process/ManagedProcess/Internal/Types.hs

```diff
@@ -287,6 +287,11 @@ instance forall s . MonadMask (GenProcess s) where
         (a', _) <- lift $ restoreP $ runProcess ourSTate p2
         return a'
 
+  generalBracket acquire release inner = GenProcess $ 
+    generalBracket (unManaged acquire)
+                   (\a e -> unManaged $ release a e)
+                   (unManaged . inner)
+
```

Comment:
All deps of this package (at least `distributed-process-supervisor` and `uplink`) break on lts-11.13 now because of `exceptions-0.10.0`. Why did you roll over to that specific version? This function `generalBracket` is not used anywhere, nor is it exported.

Reply:
Ahh, well, I'm sure lts-11.14 will have `distributed-process-0.7.4` which is what _needs_ `exceptions-0.10.0`. This is ok :+1:

Reply:
Ok, let me add backward compatibility with older exceptions. I've added that because I didn't want to introduce upper bound on the exceptions package. Also generalBracket is used in `bracket` code, and is proper solution for the state monad.

Reply:
I think the issue is that `distributed-process-0.7.4` demands `exceptions >= 0.10.0`. Since this is the case, I _think_ that means you'd have to remove `distributed-process-0.7.4` from the extra-deps, because otherwise it will compain that `exceptions >= 0.10.0` dep constraint is not met.

Reply:
If you remove both, the stack resolver `lts-11.13` will default to `distributed-process-0.7.3` and `exceptions-0.8.3`, hopefully.